aea/helpers/ipfs/utils.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2021 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+"""This module contains utility methods for ipfs helpers."""
+import contextlib
+import os
+import sys
+from typing import Generator
+
+
+@contextlib.contextmanager
+def _protobuf_python_implementation() -> Generator:
```

Comment:
Can you explain in the docstring why this is required please

Reply:
done